### PR TITLE
Add text for 'About' tab.

### LINKED ISSUE #251: Speck is missing header/descriptive text in "About" tab
>I didn't see such a level-4 header in Speck, and I thought it had the new standard layout, so that's what I used as a reference point.

_Originally posted by @mkcor in https://github.com/plotly/dash-bio/pull/245#issuecomment-474437671_

### PR DESCRIPTION
**Pull request by shammamah**
Closes #251

## About 
- [ ] This is a new component 
- [ ] I am adding a feature to an existing component, or improving an existing feature
- [x] I am closing an issue 

## Description of changes 
Change "About" tab to follow the pattern of the other redesigned apps, with a header "What is ..." and a longer description of the component/app. 

## Before merging
- [x] I have gone through the [code review checklist](https://github.com/plotly/dash-component-boilerplate/blob/master/%7B%7Bcookiecutter.project_shortname%7D%7D/review_checklist.md)
- [x] I have completed all of the [steps to take before merging](https://github.com/plotly/dash-bio/blob/master/.github/before_merging.md)
- [x] I know how to [deploy to DDS](https://github.com/plotly/dash-bio/blob/master/.github/deploying_to_dds.md)

### CODE REVIEW

File: tests/dashbio_demos/app_speck.py [hunk 1]
```diff
@@ -139,8 +139,18 @@ def layout():
...
-                        html.P(description())
-                    ])
+                        html.H4('What is Speck?'),
+                        html.P('Speck is a WebGL-based molecule renderer. By '
+                               'using ambient occlusion, the resolution of '
+                               'the rendering does not suffer as you zoom in.'),
+                        html.P('You can change the molecule to be viewed in the '
+                               '"Data" tab, and control parameters related to '
+                               'the appearance of the molecule in the "View" tab. '
+                               'These parameters can be controlled at a low level '
```
Comment (Bachibouzouk) [inline tests/dashbio_demos/app_speck.py]: I wouldn't use low and high level

```suggestion
                               'These parameters can be controlled '
```

File: tests/dashbio_demos/app_speck.py [hunk 2]
```diff
@@ -139,8 +139,18 @@ def layout():
...
+                               'with the sliders provided, or preset views can be '
```
Comment (Bachibouzouk) [inline tests/dashbio_demos/app_speck.py]: you use once "low level" and once "higher-level" below, I would adopt the same convention for both.

I would rephrase "These low-level parameters can be..."

File: tests/dashbio_demos/app_speck.py [hunk 3]
```diff
@@ -139,8 +139,18 @@ def layout():
...
+                               'applied for a higher-level demonstration of changing '
```
Comment (Bachibouzouk) [inline tests/dashbio_demos/app_speck.py]:
```suggestion
                               'applied for a high-level demonstration of changing '
```

Comment (mkcor) [reply tests/dashbio_demos/app_speck.py]: Personally I prefer @shammamah's original version (it's high*er* relative to the low level we just read about).

File: tests/dashbio_demos/app_speck.py [hunk 4]
```diff
@@ -139,8 +139,18 @@ def layout():
                     label='About',
                     value='what-is',
                     children=html.Div(className='speck-tab', children=[
-                        html.P(description())
-                    ])
+                        html.H4('What is Speck?'),
+                        html.P('Speck is a WebGL-based molecule renderer. By '
+                               'using ambient occlusion, the resolution of '
+                               'the rendering does not suffer as you zoom in.'),
+                        html.P('You can change the molecule to be viewed in the '
```
Comment (mkcor) [inline tests/dashbio_demos/app_speck.py]:
```suggestion
                        html.P('You can toggle between molecules using the menu under the '
```
Just a suggestion.
